Common/GPU/OpenGL/GLRenderManager.cpp

```diff
@@ -236,18 +236,24 @@ bool GLRenderManager::ThreadFrame() {
 			INFO_LOG(G3D, "Running first frame (%d)", threadFrame_);
 			firstFrame = false;
 		}
+
+		// Creation of OpenXR frame. This updates user's head pose and VR timestamps.
+		// For fluent rendering, the time between PreVRRender and PostVRRender must be really short.
```

Comment:
This still doesn't explain the purpose of the vrlock variable. That's what I need you to explain clearly.

Reply:
I think the variable name and also the method names were choosen bit incorrectly. I changed to make it clear.

Reply:
Yeah, that's better.